helpers.go

```diff
@@ -183,7 +183,7 @@ func (f localOptionFunc) SetLocalOption(li *llb.LocalInfo) {
 	f(li)
 }
 
-func localIncludeExcludeMerge(includes []string, excludes []string) localOptionFunc {
+func LocalIncludeExcludeMerge(includes []string, excludes []string) localOptionFunc {
```

Comment:
Same comment re: keeping this in the other PR

Also, let's look at if this needs to be exported.